examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -166,6 +166,9 @@ function implicit_tendency_special!(Yₜ, Y, p, t)
 
     ref_thermo_params = Ref(thermo_params)
     ref_zuₕ = Ref(zero(eltype(Yₜ.c.uₕ)))
+    FT = eltype(Yₜ)
+    parent(Yₜ.c.turbconv) .= FT(0)
+    parent(Yₜ.f.turbconv) .= FT(0)
```

Comment:
`.turbconv` doesn't exist by default, so you'll need a branch for when it doesn't exist.